app/views/posts/index.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Users Post</h1>
+<div class="container">
+  <%= render "shared/user_header", user: @user%>
+  <% @posts.each do |post| %>
+    <div class="w-50 border border-2 p-2">
+      <h4> <%= link_to post.title, user_post_path(post, user_id: @user.id) %> </h4>
+      <p> <%= post.text %> </p>
+      <div class="d-flex justify-content-end">
+        <span class="">Comments: <%= post.comments.count%> </span>, 
+        <span class="ml-2" > Comments: <%= post.likes.count%> </span>
+      </div>
+      <div class="border mt-1 p-2">
+        <% post.comments.each do |comment| %>
+          <p>
+            <span> <%= comment.author.name %> : </span>
+            <span> <%= comment.text %> </span>
+          </p>
+        <%end%>
+      </div>
+    </div>
+  <% end %>
+  <button class="btn btn-secondary btn-sm mt-5">Pagination comming soon</button>
```

Comment:
- [ ] Kindly center the elements to match [the wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAACBA7J7KYPXEYKZOH5JANANQZBN2FFQ).

  ![image](https://user-images.githubusercontent.com/81468675/230177519-6dd653ae-b472-4be6-bf4b-43c018572d5f.png)